KFSwiftImageLoader/MKAnnotationViewExtensions.swift

```diff
@@ -160,7 +161,7 @@ public extension MKAnnotationView {
                     }
                 }
                 
-                dataTask.resume()
+                dataTask?.resume()
```

Comment:
I'm getting errors here with beta 4. "Cannot use optional chaining on non-optional value of type 'NSURLSessionDataTask'"